### ISSUE TITLE
Sugar CRM plugin is not working as expected

### ISSUE THREAD

**Issue by toondeg** [mautic/mautic#11603]
### Mautic Version

4.3.x series

### PHP version

7.4..27

### What browsers are you seeing the problem on?

Not relevant

### What happened?

We tried to setup a connection between Mautic and sugar CRM.

How it should work:

Sugar CRM is leading sending the following data to Mautic
**First Name
Last Name
Email
Company**

Mautic is sending back one custom field
**Optin**

Outcome:
Contacts get pushed to Mautic and all fields are updated in one way, also the optin field that should go from Mautic to the CRM.

Contacts are not being synced at all, duplicates are being created.

In general it seems like the setup of the plugin is doing nothing atm, just a push from crm to mautic.

### How can we reproduce this issue?

Step 1: activate plugin
Step 2: fill in credentials
Step 3: setup contact field mapping

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by mollux** [comment]
@toondeg 

can you check if https://github.com/mautic/mautic/issues/9926 (or any other issue on https://github.com/mautic/mautic/issues?q=is%3Aissue+sugar+is%3Aclosed) describes the same problem?

Especially https://github.com/mautic/mautic/issues/9926#issuecomment-867002290 may be a similar to what you describe?

**Comment by toondeg** [comment]
Indeed, the same issue we are having as for the pushing / fetching of leads

**Comment by mollux** [comment]
@toondeg 

can you work out a detailed description on how to reproduce as detailed as possible (without confidential info)?

This would help to check if this an actual bug.

**Comment by toondeg** [comment]
Hi Mattias

Steps to reproduce:
1. enable the =Sugarcrm
2. fill in enable/ Auth
<img width="449" alt="image" src="https://user-images.githubusercontent.com/104193550/206657081-ad273119-c82f-4268-83fa-d174b317b3b2.png">
3. fill in features
<img width="453" alt="image" src="https://user-images.githubusercontent.com/104193550/206657172-d2c6a751-bbd6-4078-a641-0e8bc63d95c7.png">
4. complete contact mapping

* all user data goes from the CRM to Mautic
* Only the optin goes back to the CRM

Extra:
enabled crons:
fetchleads
synccontacts
pushactivity

**Comment by LordRembo** [comment]
bump

**Comment by LordRembo** [comment]
@RCheesley or @escopecz could this ticket be reopened? It's closed because of inactivity but still relevant.
